pkg/sysregistriesv2/system_registries_v2_test.go

```diff
@@ -383,6 +383,12 @@ func TestFindRegistry(t *testing.T) {
 	assert.Equal(t, "empty-prefix.com", reg.Prefix)
 	assert.Equal(t, "empty-prefix.com", reg.Location)
 
+	reg, err = FindRegistry(sys, "empty-location.set-prefix.example.com/namespace/repo")
+	assert.Nil(t, err)
+	assert.NotNil(t, reg)
```

Comment:
(Non-blocking: These should be `require.`. OTOH this is consistent with the rest, and only matters on test failures.)

Reply:
I honestly don’t think this is worth much worrying about; if you find updating pre-existing test cases fun, sure, go for it, but it’s not work that results in anything users could notice and consider valuable while _using_ the software. As far as “clean code” goes, `require` is a bit more correct but consistency is also nice, so to me that’s not a strong reason to prefer either.

(Just to be explicit: `require` should be used for testing prerequisites that must be true for other checks to be even valid to do, e.g.
```go
ptr, err := testSubject()
require.NoError(t, err)
assert.Equal(t, …, *ptr) // would crash if testSubject returned (nil, err) and the above were assert.NoError(…)
```

But there is a _minimal_ practical difference; with `assert.NoError(t, err)` the test crashes with a backtrace, with `require.NoError(t, err)` the test fails with an error message containing `err`. It doesn’t matter at all if the test completely succeeds. If the test fails because `err` is set, in both cases the test fails, it’s easy enough to identify the location of the failure, and probably something about `testSubject` needs to be updated. `require` is only better in that it includes the actual error text in the test failure output, and that could help with a inconsistent rarely-occurring failure in CI.)